src/app.js

```diff
@@ -15,17 +15,16 @@ app.get('/', (request, response) => {
   response.send('Place holder');
 });
 
-app.get('/api/v1/songs', (request, response) => {
-  database('songs').select()
-    .then((papers) => {
-      response.status(200).json(papers);
+app.get('/api/v1/favorites', (request, response) => {
+  database('songs').select(["name", "artist_name", "genre", "song_rating"])
+    .then((favorite) => {
+      response.status(200).json({ name: favorite[0], artist_name: favorite[1], genre: favorite[2], song_rating: favorite[3] });
```

Comment:
Interesting, perhaps there is a better way to serialize this data? Something to look into! For now I'm merging, great job Jesse!